File: pinyougou-interface/src/main/java/cn/itcast/core/service/orderd/OrderdService.java
package cn.itcast.core.service.orderd;

import cn.itcast.core.pojo.orderd.Orderd;

import java.util.List;

public interface OrderdService {
    //查询所有order
    List<Orderd> findAll();

    //新增order
    void add(Orderd orderd);

    //取消（删除）order
    public void delete(Long[] ids);

    //修改order

    void update(Orderd orderd);

}

File: pinyougou-web-shop/src/main/webapp/js/service/orderdService.js
//服务层
app.service('orderdService',function($http){
	    	
	//查询
	this.findAll=function(){
		return $http.get('../orderd/findAll.do');		
	}
	//分页
	this.findPage=function(page,rows){
		return $http.get('../orderd/findPage.do?page='+page+'&rows='+rows);
	}

	//增加 
	this.add=function(entity){
		return  $http.post('../orderd/add.do',entity );
	}

	//删除
    this.dele = function(ids){
        return $http.get("../orderd/delete.do?ids="+ids);
    }

    //修改 
    this.update=function(entity){
        return  $http.post('../orderd/update.do',entity );
    }
});
